### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by majst01**
Fix all typos found with https://github.com/crate-ci/typos, later we can add this to the CI to prevent further introduction of typos.

- [x] read the [CONTRIBUTING guidelines](README.md#contributing)
- [ ] raised a GitHub issue or discussed it on the projects chat beforehand
- [ ] added unit tests
- [ ] added integration tests
- [ ] updated documentation if needed
- [ ] updated CHANGELOG.md

### CODE REVIEW

**Review by ohdearaugustin — APPROVED** [review]
Looks good and shouldn't change any code logic.

**Comment by ohdearaugustin** [comment]
Please run prettier on docs/web-ui.md

**Comment by majst01** [comment]
> Please run prettier on docs/web-ui.md

done

**Review by ohdearaugustin — APPROVED** [review]

**Comment by ohdearaugustin** [comment]
There have been a lot of changes can you please rebase the PR.

**Comment by majst01** [comment]
> There have been a lot of changes can you please rebase the PR.

Done

**Comment by ohdearaugustin** [comment]
There are still some conflicts.

File: docs/examples/kustomize/base/secrets/private-key
Comment (ohdearaugustin) [inline docs/examples/kustomize/base/secrets/private-key:1]: I think you merge a local main and your secrets.

File: docs/examples/kustomize/base/site/public.env
Comment (ohdearaugustin) [inline docs/examples/kustomize/base/site/public.env:1]: I think you merge a local main and your env

File: docs/examples/kustomize/postgres/secrets/password
Comment (ohdearaugustin) [inline docs/examples/kustomize/postgres/secrets/password:1]: Another secret

**Review by ohdearaugustin — CHANGES_REQUESTED** [review]
Delete your secrets from the kustomize

**Comment by kradalby** [comment]
I think we should just remove the whole kustomize stuff, I can open a pr for it tomorrow

**Comment by kradalby** [comment]
@ohdearaugustin opened for removing kustomize #1906

**Comment by majst01** [comment]
Should be in a sane state finally :-)

**Comment by majst01** [comment]
synced with main again

**Review by ohdearaugustin — APPROVED** [review]

**Comment by ohdearaugustin** [comment]
The unstable tailscale client is failing for the IntegrationTest (TestAuthKeyLogoutAndRelogin) this was not caused by this PR, but rather by an update of the client. @kradalby I hope it is okay to merge it.

**Comment by kradalby** [comment]
Yes it's been in the works for a bit, great getting it in

**Comment by majst01** [comment]
Thanks

**Pull request merged**